docs/backups-restore.md

```diff
@@ -1,34 +1,56 @@
 # Restore the cluster from a previously saved backup
 
-The backup is normally restored on the Kubernetes cluster where it was made, but [restoring it on a different Kubernetes-based environment with the installed Operator is also possible](backups-restore-to-new-cluster.md).
+You can restore from a backup as follows:
 
-Following things are needed to restore a previously saved backup:
+* On the same cluster where you made a backup
+* On [a new cluster deployed in a different Kubernetes-based environment](backups-restore-to-new-cluster.md).
 
-* Make sure that the cluster is running.
-* Find out correct names for the **backup** and the **cluster**. Available backups can be listed with the following command:
+This document focuses on the restore to the same cluster.
+
+To restore from a backup, you create a Restore object using a special restore configuration file. The
+example of such file is [deploy/backup/restore.yaml :octicons-link-external-16:](https://github.com/percona/percona-server-mongodb-operator/v{{ release }}/deploy/backup/restore.yaml).
+
+You can check available options in the [restore options reference](restore-options.md).
+
+## Restore scenarios
+
+This document covers the following restore scenarios:
+
+* [Restore from a full backup](#restore-from-a-full-backup) - restore from a backup snapshot without point-in-time
+* [Point-in-time recovery](#restore-with-point-in-time-recovery) - restore to a specific time, a specific or latest transaction or skip a specific transaction during a restore. This ability requires that you [configure storing oplog for point-in-time recovery](backups-pitr.md)
+* [Selective restore from a full logical backup](#selective-restore)
+
+--8<-- [start:backup-prepare]
+
+## Before you start
+
+1. Make sure that the cluster is running.
+2. List the cluster to find the correct cluster name. Replace the `<namespace>` with your value:
 
     ``` {.bash data-prompt="$" }
-    $ kubectl get psmdb-backup
+    $ kubectl get psmdb -n <namespace>
     ```
 
-    And the following command will list available clusters:
+3. List backups to retrieve the desired backup name. Replace the `<namespace>` with your value:
 
     ``` {.bash data-prompt="$" }
-    $ kubectl get psmdb
+    $ kubectl get psmdb-backup -n <namespace>
     ```
 
-!!! note
+--8<-- [end:backup-prepare]
+
+## Restore from a full backup
 
-     If you have [configured storing operations logs for point-in-time recovery](backups-pitr.md), you will have possibility to roll back the cluster to a specific date and time. Otherwise, restoring backups without point-in-time recovery is the only option.
+To restore your Percona XtraDB cluster from a backup, define a `PerconaServerMongoDBRestore` custom resource. Set the following keys:
```

Comment:
this should say Percona Server for MongoDB cluster or something like that, not PXC